aptos-move/block-executor/src/executor.rs

```diff
@@ -343,14 +328,7 @@ where
             use KeyKind::*;
             match kind {
                 Resource => versioned_cache.data().remove(&k, idx_to_execute),
-                Module => {
```

Comment:
do we use this kind anywhere? if not we can get rid of it as well (o.w. we can add a comment that modules are published in sequential commit hook)

Reply:
Good catch! We do not, removed 👍